spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -49,9 +49,12 @@
 import org.apache.spark.sql.types.Metadata;
 import org.apache.spark.sql.types.StructField;
 import org.apache.spark.sql.types.StructType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 class AddFilesProcedure extends BaseProcedure {
 
+  private static Logger LOG = LoggerFactory.getLogger(AddFilesProcedure.class);
```

Comment:
I think checkstyle is not happy about variable name, 
```
Error: eckstyle] [ERROR] /home/runner/work/iceberg/iceberg/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java:57:25: Name 'LOG' must match pattern '^[a-z][a-zA-Z0-9]*$'. [StaticVariableName]
```

maybe `logger` instead?

Reply:
yes, should have been `log`. Fixed thanks.